modules/local/export_meta.nf

```diff
@@ -23,6 +13,9 @@ process EXPORT_META {
     output:
     path "*.csv", emit: csv
 
```

Comment:
version output is missing here

Reply:
there is no version, its just writing the meta data to file